src/app/cart/cart.component.js

```diff
@@ -60,8 +60,17 @@ class CartController {
     this.$window.location.href = this.sessionService.getRole() === 'REGISTERED' ? 'checkout.html' : 'sign-in.html';
   }
 
-  donationStartDate( month, day ) {
-    return new Date( 0, Number( month ) - 1, Number( day ) );
+  donationStartDate( config ) {
```

Comment:
It is part of the cart service.  This code it pulling it and showing it.  I was wondering if you had any feedback on what I came up with to prevent digest loops.